_includes/auth/gen-rsa-key-pairs.md

```diff
@@ -65,15 +65,13 @@ sudo zypper install openssl
 
 To create a private/public key pair:
 
-1. On a command line, run:
+On a command line, run:
```

Comment:
```suggestion
On a command line run the following, where _`<MY_KEY_PAIR_NAME>`_ should be replaced with a name for the key pair:
```
instead of splitting the sentence?

Reply:
I wonder if we need even more than this? As having the `To create` thing above makes it look like we're going to mention a list but then we don't :shrug:

Reply:
Whatever the decision was, it was the right one, 'cos it looks good